sandbox/lua/encoders/schema_influx_line.lua

```diff
@@ -0,0 +1,165 @@
+    dynamic message field can be used for name name interpolation. If the
+    dynamic field doesn't exist, the uninterpolated value will be left in the
+    name. Note that it is not possible to interpolate either the
+    "Timestamp" or the "Uuid" message fields into the name, those
+    values will be interpreted as referring to dynamic message fields.
+
+- name_prefix_delimiter (string, optional, default nil)
+    String to use as the delimiter between the name_prefix and the field
+    name.  This defaults to a blank string but can be anything else
+    instead (such as "." to use Graphite-like naming).
+
+- skip_fields (string, optional, default nil)
+    Space delimited set of fields that should *not* be included in the
+    InfluxDB measurements being generated. Any `fieldname` values of "Type",
+    "Payload", "Hostname", "Pid", "Logger", "Severity", or "EnvVersion" will
+    be assumed to refer to the corresponding field from the base message
+    schema. Any other values will be assumed to refer to a dynamic message
+    field. The magic value "**all_base**" can be used to exclude base fields
+    from being mapped to the event altogether (useful if you don't want to
+    use tags and embed them in the name_prefix instead).
+
+- source_value_field (string, optional, default nil)
+    If the desired behavior of this encoder is to extract one field from the
+    Heka message and feed it as a single line to InfluxDB, then use this option
+    to define which field to find the value from.  Be careful to set the
+    name_prefix field if this option is present or no measurement name
+    will be present when trying to send to InfluxDB.  When this option is
+    present, no other fields besides this one will be sent to InfluxDB as
+    a measurement whatsoever.
+
+- tag_fields (string, optional, default "**all_base**")
+    Take fields defined and add them as tags of the measurement(s) sent to
+    InfluxDB for the message.  The magic values "**all**" and "**all_base**"
+    are used to map all fields (including taggable base fields) to tags and only
+    base fields to tags, respectively.  If those magic values aren't used,
+    then only those fields defined will map to tags of the measurement sent
+    to InfluxDB. The tag_fields values are independent of the skip_fields
+    values and have no affect on each other.  You can skip fields from being
+    sent to InfluxDB as measurements, but still include them as tags.
+
+- timestamp_precision (string, optional, default "ms")
+    Specify the timestamp precision that you want the event sent with.  The
+    default is to use milliseconds by dividing the Heka message timestamp
+    by 1e6, but this math can be altered by specifying one of the precision
+    values supported by the InfluxDB write API (ms, s, m, h). Other precisions
+    supported by InfluxDB of n and u are not yet supported.
+
+- value_field_key (string, optional, default "value")
```

Comment:
I decided to change the name of this field based on InfluxDB vocabulary in their documentation.